uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkDayTest.php

```diff
@@ -183,6 +183,37 @@ public function testGetWeekendTypeValueReturnsTheOptionValueValueAsString() {
     $this->assertSame((string)$workDayTypes['weekend'], WorkDay::getWeekendTypeValue());
   }
 
+  public function testCreateWorkDayWithLeaveDaysFromLeaveDaysAmountOptionGroup() {
```

Comment:
This test is too complex and in the end it's not clear what we're really testing. To make it simpler, we can focus on what is really important, which is saving decimal amounts.

We can just check that it can save different decimals amounts like 0.34, 1.5, 1.25 etc (just a few variations are enough)